### PR TITLE
Daniela/pokerhands

### PR DESCRIPTION
**Pull request by DaniMB95**
Create almost all the games for a pokerhand and create a class enum to rank the games of a pokerhand and I will keep continue working in the way of identify a game.

### CODE REVIEW

**Review by J0c4 — APPROVED** [review]

**Comment by DaniMB95** [comment]
In the last commit I only implement the gameanalizer file to onli determine what game it's the pokerhand and also create the winnin file to see who wins in two pokerhands. 
I see how sorted the hand but I'mstill working on it and also still working on  determine who wins in a tie

**Comment by DaniMB95** [comment]
@J0c4 I take a other way to analyze the highcard in a tie so I create two new classes the CardstoString class where I only put inside the score of every card and inside that class I take the score of the summatory of all cards score and the other class I create was the Cardssorted where I can analize a pokerhand and take the string of the cards.
Finally I refactor the winning class to determine the winner in a tie, but I need to refactor to follow the rules.

**Comment by DaniMB95** [comment]
Add the class  Cardssortedfinal and the class Cardssort where I sort the cards of a pokerhand in a ascending format

File: src/main/java/jalau/at18/katas/pokerhands/daniela/CardValue.java [hunk 1]
```diff
@@ -1,27 +1,37 @@
...
+    CardValue(CardValue next, int value) {
         this.next = next;
+        this.valuecard = value;
     }
 
     public CardValue next() {
         return next;
     }
+
+    public int getRealNumberValue() {
```
Comment (J0c4) [inline src/main/java/jalau/at18/katas/pokerhands/daniela/CardValue.java:30]: This should be removed if it is not needed anymore

File: src/main/java/jalau/at18/katas/pokerhands/daniela/CardValuesCount.java [hunk 2]
```diff
@@ -1,6 +1,8 @@
 package jalau.at18.katas.pokerhands.daniela;
 
+//import java.util.ArrayList;
 import java.util.HashMap;
+//import java.util.List;
```
Comment (J0c4) [inline src/main/java/jalau/at18/katas/pokerhands/daniela/CardValuesCount.java:5]: If no reason to keep them, lets remove commented code

File: src/main/java/jalau/at18/katas/pokerhands/daniela/FourofaKind.java [hunk 3]
```diff
@@ -1,28 +1,21 @@
...
-        this.value1 = value1;
-        this.value2 = value2;
+    public FourofaKind() {
     }
 
     @Override
     public boolean match(PokerHand handWith4SimilarCards) {
-        onePairs(handWith4SimilarCards);
-        return onePairs(handWith4SimilarCards);
+        fourKind(handWith4SimilarCards);
```
Comment (J0c4) [inline src/main/java/jalau/at18/katas/pokerhands/daniela/FourofaKind.java:12]: This line can be removed

**Review by J0c4 — CHANGES_REQUESTED** [review]

File: src/main/java/jalau/at18/katas/pokerhands/daniela/CardValue.java (hunk 1 above)
Comment (DaniMB95) [reply src/main/java/jalau/at18/katas/pokerhands/daniela/CardValue.java:30]: Already removed

File: src/main/java/jalau/at18/katas/pokerhands/daniela/CardValuesCount.java (hunk 2 above)
Comment (DaniMB95) [reply src/main/java/jalau/at18/katas/pokerhands/daniela/CardValuesCount.java:5]: Already deleated all the comments in the code.

File: src/main/java/jalau/at18/katas/pokerhands/daniela/FourofaKind.java (hunk 3 above)
Comment (DaniMB95) [reply src/main/java/jalau/at18/katas/pokerhands/daniela/FourofaKind.java:12]: already removed the line in the classes that was duplicated

**Review by J0c4 — APPROVED** [review]

**Pull request merged**